### PR TITLE
MGMT-16494: Move ip hint file creation to ignition in order to change it in IBI process

### PR DESCRIPTION
**Pull request by tsorya**
## List all the issues related to this PR

- [ ] New Feature 
- [x] Enhancement 
- [ ] Bug fix
- [ ] Tests
- [ ] Documentation
- [ ] CI/CD 

## What environments does this code impact?

- [ ] Automation (CI, tools, etc)
- [ ] Cloud
- [ ] Operator Managed Deployments
- [x] None

## How was this code tested?

- [x] assisted-test-infra environment
- [ ] dev-scripts environment
- [ ] Reviewer's test appreciated
- [x] Waiting for CI to do a full test run
- [ ] Manual (Elaborate on how it was tested)
- [ ] No tests needed

## Checklist

- [ ] Title and description added to both, commit and PR.
- [ ] Relevant issues have been associated (see [CONTRIBUTING] guide)
- [ ] This change does not require a documentation update (docstring, `docs`, README, etc)
- [ ] Does this change include unit-tests (note that code changes require unit-tests)

## Reviewers Checklist

- Are the title and description (in both PR and commit) meaningful and clear?
- Is there a bug required (and linked) for this change?
- Should this PR be backported?

[Kubernetes community documentation]: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#commit-message-guidelines
[CONTRIBUTING]: https://github.com/openshift/assisted-service/blob/master/CONTRIBUTING.md

### CODE REVIEW

File: internal/ignition/ignition.go [hunk 1]
```diff
@@ -1510,7 +1511,7 @@ func (g *installerGenerator) modifyPointerIgnitionMCP(poolName string, ignitionS
 	return "", errors.Errorf("machine config pool %s was not found", poolName)
 }
 
-func (g *installerGenerator) writeSingleHostFile(host *models.Host, baseFile string, workDir, serviceBaseURL, bootReporter string, authType auth.AuthType) error {
+func (g *installerGenerator) writeSingleHostFile(host *models.Host, baseFile string, workDir string) error {
```
Comment (eranco74) [inline internal/ignition/ignition.go:1514]: Any clue why `serviceBaseURL, bootReporter and  authType` are in the signature???

**Review by eranco74 — COMMENTED** [review]
/LGTM

File: internal/ignition/ignition.go (hunk 1 above)
Comment (tsorya) [reply internal/ignition/ignition.go:1514]: Were not used so i removed them

**Comment by romfreiman** [comment]
/hold
Lets discuss whether the clusters will be different

**Comment by eranco74** [comment]
/lgtm
@tsorya we should check if this needs to be backported

**Comment by romfreiman** [comment]
also abi

**Comment by tsorya** [comment]
/override ci/prow/e2e-agent-compact-ipv4
test is broken

**Pull request merged**